lib/source-code/source-code.js

```diff
@@ -962,6 +962,15 @@ class SourceCode extends TokenStore {
         varsCache.set("configGlobals", configGlobals);
     }
 
+    static parseDirective(commentValue) {
```

Comment:
`SourceCode` class is part of the public API. Perhaps we could rather add this in `lib/linter/config-comment-parser.js`?

Reply:
Definitely! I'll do that.